contracts/src/tests/test_world.cairo

```diff
@@ -177,6 +177,32 @@ mod tests {
         actions_system.move(mancala_game.game_id, selected_pit);
     }
 
+    #[test]
+    #[should_panic(expected: ("Game is not in progress", 0x454e545259504f494e545f4641494c4544))]
+    fn test_cannot_move_if_game_timeout() {
+        let (_, _, world, actions_system, mut mancala_game, _) = setup_game();
+        mancala_game.status = GameStatus::TimeOut;
+        // the below line should panic
+        set!(world, (mancala_game));
+        let selected_pit: u8 = 1;
+        actions_system.move(mancala_game.game_id, selected_pit);
+    }
+
+    #[test]
+    #[should_panic(expected: ("Game is in progress", 0x454e545259504f494e545f4641494c4544))]
+    fn test_cannot_call_timeout_if_move_is_allowed() {
+        // test that the seed should go in mancala and current player should remain the same
+        let (_, _, _, actions_system, game, _) = setup_game();
+        actions_system.time_out(game.game_id);
+    }
+
+    #[test]
+    fn test_can_call_timeout_once_enough_time_has_passed() {
+        // test that the seed should go in mancala and current player should remain the same
+        let (_, _, _, actions_system, game, _) = setup_game();
+        set_block_number(10000);
+        actions_system.time_out(game.game_id);
```

Comment:
test looks great but we need to assert that the status of the game is GameStatus::TimeOut and the winner is the opponent.

Reply:
yes true thanks!